python/ray/serve/tests/test_deploy.py

```diff
@@ -63,11 +61,30 @@ def d(*args):
     assert pid5 != pid4
 
 
+def test_empty_decorator(serve_instance):
+    @serve.deployment
+    def func(*args):
+        return "hi"
+
+    @serve.deployment
+    class Class:
+        def ping(self, *args):
+            return "pong"
+
+    func._name = "func"
+    Class._name = "Class"
```

Comment:
What are these lines for?

Reply:
oops lol these are supposed to be an assert